src/itest/java/uk/gov/companieshouse/pscdataapi/steps/PscDataSteps.java

```diff
@@ -0,0 +1,97 @@
+package uk.gov.companieshouse.pscdataapi.steps;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.assertj.core.api.Assertions;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.companieshouse.pscdataapi.config.AbstractMongoConfig.mongoDBContainer;
+
+import uk.gov.companieshouse.pscdataapi.config.CucumberContext;
+import uk.gov.companieshouse.pscdataapi.util.FileReaderUtil;
+import uk.gov.companieshouse.pscdataapi.repository.CompanyPscRepository;
+
+import java.io.IOException;
+import java.util.Collections;
+
+public class PscDataSteps {
+    private String contextId;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private TestRestTemplate restTemplate;
+
+    @Autowired
+    private MongoTemplate mongoTemplate;
+
+    @Autowired
+    private CompanyPscRepository companyPscRepository;
+
+    private final String companyNumber = "34777772";
```

Comment:
private final String varaibles are typically in UPPER_CASE_SNAKE_CASE
```suggestion
    private final String COMPANY_NUMBER = "34777772";
```